.gitignore

```diff
@@ -14,6 +14,7 @@ coverage.json
 cache/
 yarn*
 frontend/
+.env
```

Comment:
+1 for using dotenv, we should remove the .private.json file though as we now have 2 files with configs